server-phoenix/lib/helios_web/schema/helpers/location.ex

```diff
@@ -0,0 +1,33 @@
+defmodule HeliosWeb.Schema.Helpers.Location do
+  def time_now(time_zone) do
+    DateTime.now!(time_zone)
+  end
+
+  def sunset(dt, lat, long, offset \\ nil) do
```

Comment:
Nit: could this default to 0 instead to avoid the branching?

Also, could you add some unit tests? That'll help resolve the sunrise/sunset differences with the ruby gem.

Reply:
Oh yeah, I'll change it to default to 0. I'll also start a PR for unit tests once this one is merged.